### PR TITLE
Update core.py

### PR DESCRIPTION
**Pull request by mikesokolsky**
Checking out the PR tool, ignore this

### CODE REVIEW

File: core.py [hunk 1]
```diff
@@ -31,7 +32,7 @@ def _request_tle(self):
                   })
 
     def update_tle(self):
-        response = self._request_tle()
+        response = self._request_stle()
```
Comment (mikesokolsky) [inline core.py]: Not quite sure what an stle is?

File: core.py [hunk 2]
```diff
@@ -54,9 +55,9 @@ def __init__(self, latitude=None, longitude=None, altitude=0):
         always default to 0 unless specified.
         Latitude is positive North
         Longitude is positive East
+        Introduced a bug
```
Comment (mikesokolsky) [inline core.py]: Yep.

**Review by mikesokolsky — COMMENTED** [review]
Plz review @mdevs5531

File: core.py (hunk 1 above)
Comment (mdevs5531) [reply core.py]: I agree.  Should be ttle.

File: core.py (hunk 2 above)
Comment (mdevs5531) [reply core.py]: +1

Comment (mdevs5531) [inline core.py]: What is this? A new conversation?

**Review by mdevs5531 — CHANGES_REQUESTED** [review]
Fixes needed.

File: core.py (hunk 1 above)
Comment (mikesokolsky) [reply core.py]: ok, fixing

**Review by mdevs5531 — APPROVED** [review]

**Pull request closed without merging**